specification/containerinstance/resource-manager/Microsoft.ContainerInstance/2017-08-01-preview/containerInstance.json

```diff
@@ -41,7 +41,8 @@
             "$ref": "./examples/ContainerGroupsList.json"
           }
         },
-        "description": "Get the list of container groups in a given subscription.",
+        "summary": "Get a list of container groups in the specified subscription.",
```

Comment:
Added summary fields on each explicit rest call to conform to how ACS gens docs.